tech/languages/rust/rust-installation.md

```diff
@@ -9,10 +9,25 @@ description: Systems programming language that runs blazingly fast, prevents seg
 
 # Rust installation
 
-In order to get started with Rust programming, type:
+## Prepackaged Rust
```

Comment:
```suggestion
## Rust
```
IMO either just Rust or Packaged Rust or maybe Fedora Distributed Rust? Prepackaged sounds a bit weird to me.